src/archivematicaCommon/lib/fileOperations.py

```diff
@@ -116,9 +116,7 @@ def writeToFile(output, fileName, writeWhite=False):
 
 def renameAsSudo(source, destination):
     """Used to move/rename Directories that the archivematica user may or may not have writes to move"""
-    command = "sudo mv \"" + source + "\"   \"" + destination + "\""
-    if isinstance(command, unicode):
-        command = command.encode("utf-8")
+    command = ["sudo", "mv", source, destination]
```

Comment:
Testing, no - mixed encodings are fine if no shell splitting is taking place. e.g. this is valid, with one unistring and one bytestring that's not valid Unicode:

```python
subprocess.check_output(['echo', u'é', '\xFF'])
```